tests/test_define.h

```diff
@@ -332,6 +332,8 @@ extern "C" {
 #define MESSAGE_5 "MESSAGE5"
 #define MESSAGE_6 "MESSAGE6"
 
+#define TEST_PROXY_API_addNeighbors "addNeighbors"
```

Comment:
Use the technique of Concatenation in C preprocessor: https://gcc.gnu.org/onlinedocs/cpp/Concatenation.html
We can elegantly rewrite the code snip as following:
```c
#define TEST_PROXY_API(cmd) #cmd
```

Then, `TEST_PROXY_API(addNeighbors)` is equivalent to `"addNeighbors"`.